docs/content/docs/build/audience.md

```diff
@@ -36,17 +37,17 @@ export interface IMember {
 }
 ```
 
-The `IMember` represents a single user identity.  The `IMember` holds a list of `IConnection`s, which represent that audience member's active connections to the container.  Typically a user will only have one connection, but scenarios such as loading the container in multiple web contexts or on multiple computers will also result in multiple connections.  An audience member will always have at least one connection.  Each user and each connection will both have a unique indentifier.
+An `IMember` represents a single user identity.  `IMember` holds a list of `IConnection`s, which represent that audience member's active connections to the container.  Typically a user will only have one connection, but scenarios such as loading the container in multiple web contexts or on multiple computers will also result in as many connections.  An audience member will always have at least one connection.  Each user and each connection will both have a unique indentifier.
 
 {{% callout tip %}}
 
-Connections are frequently short-lived and not reused - a client that disconnects from the container and immediately reconnects will receive an entirely new connection.  The audience will reflect this change, and your feature may need to account for this scenario.
+Connections can be short-lived and are not reused - a client that disconnects from the container and immediately reconnects will receive an entirely new connection.  The audience will reflect this as a member leaving and a member joining.
 
 {{% /callout %}}
 
 ### Service-Specifics
 
-The `ServiceAudience` class represents the base audience implementation, and individual services are expected to extend this class for their needs.  Typically this is through extending `IMember` to provide richer user information.  For `TinyliciousAudience`, this is the only change, and it defines a `TinyliciousMember` to add a user name.  `TinyliciousAudience` then provides `TinyliciousMember`s instead of plain `IMember`s.
+The `ServiceAudience` class represents the base audience implementation, and individual services are expected to extend this class for their needs.  Typically this is through extending `IMember` to provide richer user information and then extending `ServiceAudience` to use the `IMember` extension.  For `TinyliciousAudience`, this is the only change, and it defines a `TinyliciousMember` to add a user name.
```

Comment:
i think this one should stay as is because the suggestions are a functional change.  ServiceAudience is an abstract class so it must be extended.  extending IMember is the only required extension, but they can do other things too which i didn't think were worth mentioning here (like change client exclusions).